src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Paste.java

```diff
@@ -52,46 +53,64 @@ public void run() {
-                final Paste paste = new Paste(plugin);
-                final String pasteText = paste.genNewPaste();
-                String pasteResult = paste.paste(pasteText);
-                if (pasteResult != null) {
-                    //MsgUtil.sendMessage(sender, pasteResult);
-                    // So we send direct message to sender to make sure our paste system still can working if MsgUtil down.
-                    sender.sendMessage(pasteResult);
-                    plugin.log(pasteResult);
-                } else {
-                    MsgUtil.sendMessage(sender, "The paste failed, saving the paste at local location...");
-                    File file = new File(plugin.getDataFolder(), "paste");
-                    file.mkdirs();
-                    file =
-                            new File(file, "paste-" + UUID.randomUUID().toString().replaceAll("-", "") + ".txt");
-                    try {
-                        boolean createResult = file.createNewFile();
-                        Util.debugLog("Create paste file: " + file.getCanonicalPath() + " " + createResult);
 
-                        try (FileWriter fwriter = new FileWriter(file)) {
-                            fwriter.write(pasteText);
-                            fwriter.flush();
-                        }
-                        MsgUtil.sendMessage(sender, "Paste was saved to your server at: " + file.getAbsolutePath());
-                    } catch (IOException e) {
-                        plugin.getSentryErrorReporter().ignoreThrow();
-                        e.printStackTrace();
-                        MsgUtil.sendMessage(sender, "Saving failed, output to console...");
-                        plugin.getLogger().info(pasteText);
+                    if (Arrays.stream(cmdArg).anyMatch(str -> str.contains("--file"))) {
+                        pasteToLocalFile(sender);
+                        return;
                     }
+
+                    if (!pasteToPastebin(sender)) {
+                        pasteToLocalFile(sender);
+                    }
+
                 }
+                MsgUtil.sendMessage(sender, "§aPlease wait, we're uploading the data to the pastebin...");
             }
         }.runTaskAsynchronously(plugin);
     }
 
+    private boolean pasteToPastebin(@NotNull CommandSender sender) {
+        MsgUtil.sendMessage(sender, "§aPlease wait, we're uploading the data to the pastebin...");
+        final Paste paste = new Paste(plugin);
+        final String pasteText = paste.genNewPaste();
+        String pasteResult = paste.paste(pasteText);
+        if (pasteResult != null) {
+            sender.sendMessage(pasteResult);
+            plugin.log(pasteResult);
+            return true;
+        }
+        return false;
+    }
+
+    private boolean pasteToLocalFile(@NotNull CommandSender sender) {
+        MsgUtil.sendMessage(sender, "The paste failed, saving the paste at local location...");
```

Comment:
This message should be send in line 68, otherwise paste to local file will always see this.